myning/objects/research_facility.py

```diff
@@ -112,15 +112,16 @@ def points_per_hour(self):
         ticks_per_hour = 60 / self.minutes_per_tick
         return ticks_per_hour * self.points_per_researcher * len(self._researchers)
 
-    def check_in(self):
+    def check_in(self, bonus: int = 1):
         if not self.last_research_tick:
             self.last_research_tick = datetime.now()
             return
 
         mins_since_last_tick = (datetime.now() - self.last_research_tick).total_seconds() / 60
         tick_completion = mins_since_last_tick / self.minutes_per_tick
 
-        self._points += self.points_per_researcher * len(self._researchers) * tick_completion
+        earned = self.points_per_researcher * len(self._researchers) * tick_completion
+        self._points += earned * bonus
```

Comment:
```suggestion
        self._points += self.points_per_researcher * len(self._researchers) * tick_completion * bonus
```

Reply:
Linter forces is to three lines anyhow, so I figured this was more clean than the parentheses wrapping it.